### DISCUSSION TITLE
More than one Light-Device?

### DISCUSSION THREAD

**Discussion by Battlecake91** [arendst/Tasmota#18338] [Q&A]
Hello everyone,

I'm trying to build up some stuff with Tasmota. 

Most of the things are working well, but I'm struggling a bit with some things I thought it should be very easy.

My situation: I try to use an ESP32-C3 for SK6812 (works great) AND 1 PWM Channel for a white LED. 
But if I config Tasmota for WS28xx and one PWM channel, I only have one controlable channel. I thought there will be something like Power1 and Power2, but the console says Power2 is unkown.

Is there any way to get SK6812 AND a PWM channel working? 

Greetings

**Comment by sfromis** [comment]
You cannot combine PWM and neopixel for two lights, unless you code at least one in Berry, to manage details yourself.
